src/mumble/Log.cpp

```diff
@@ -728,12 +724,28 @@ void Log::log(MsgType mt, const QString &console, const QString &terse, bool own
 		const int oldscrollvalue = tlog->getLogScroll();
 		const bool scroll        = (oldscrollvalue == tlog->getLogScrollMaximum());
 
+		// A newline is inserted after each frame, but this spaces out the
+		// log entries too much, so the line height is set to zero to reduce
+		// the space between log entries. This line height is only set for the
+		// blank lines between entries, not for entries themselves.
+		//
+		// NOTE: All log entries must go in a text frame. Otherwise, they will
+		// not display correctly as a result of having line height equal to 0.
```

Comment:
```suggestion
		// NOTE: All further log entries must go in new a text frame. Otherwise, they will
		// not display correctly as a result of having line height equal to 0 for the current block.
```

Reply:
Done